codegen-core/src/main/kotlin/io/vrap/rmf/codegen/types/VrapType.kt

```diff
@@ -28,6 +28,31 @@ open class VrapObjectType(val `package` :String, val simpleClassName:String) : V
     }
 }
 
+class VrapDateTimeType(`package` :String, simpleClassName:String, val format: String): VrapObjectType(`package`,simpleClassName) {
```

Comment:
Here i have a feeling that only format makes sense, since once you know it you translate the date time to a proper standard type, also the format should be an enum since we already know all the values it can take.

Reply:
I added an enum for DateTimes

Reply:
Sorry a question, i don't see why the package is necessary here https://github.com/vrapio/rmf-codegen/pull/148/commits/0cabd95f6e74b1ded319e8ff820586dda526f199#diff-9c66eb3cea406ceb052e1ba0058f4a1dR37

Reply:
The date classes are living in the global namespace for PHP. I don't know if this is the case for all languages? Maybe you want to use in some generate another implementation.

Reply:
Cause I use the DateTime types for specialized serialization which converts to UTC and prints to specific format. Don't know how you are doing this in other generators. But this could come handy.

Reply:
To be fair so far i didn't check on other languages, but i would expect java to have a similar problem with different date formats. now i see what you did this and it makes sense to me 👍